src/main/java/com/thealgorithms/maths/Average.java

```diff
@@ -25,15 +31,15 @@ public static double average(double[] numbers) {
     }
 
     /**
-     * find average value of an int array
+     * Computes the average of an {@code int} array.
      *
-     * @param numbers the array contains element and the sum does not excess long
-     *                value limit
-     * @return average value
+     * @param numbers an array of {@code int} values
+     * @return the average of the given numbers
+     * @throws IllegalArgumentException if the input array is {@code null} or empty
      */
     public static int average(int[] numbers) {
```

Comment:
```suggestion
    public static double average(int[] numbers) {
```
Change the data type

Reply:
Roger, applied